### PR TITLE
JN-478 updated admin ui header

### PR DESCRIPTION
**Pull request by devonbush**
This adds a help link in the top right, and replaces the user name with a user icon.  Goal is to move towards the invision prototype here https://dspuxteam.invisionapp.com/overview/Juniper---Nav-Concept-clk06hf7d2iiw01ah8a517ir0/screens?utm_source=email&sortBy=1&sortOrder=1&viewLayout=2

Nav look
image

Help popup
image

TO TEST:
1. go to admin tool
2. click around the top-right menu, confirm stuff still works

### CODE REVIEW

File: ui-admin/src/navbar/AdminNavbar.tsx [hunk 1]
```diff
@@ -56,14 +59,36 @@ function AdminNavbar({ breadCrumbs, sidebarContent, showSidebar, setShowSidebar
...
           <ul className="navbar-nav ms-auto">
+            <li className="nav-item dropdown">
+              <a className="nav-link text-white" href="#"
+                role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                <FontAwesomeIcon icon={faQuestionCircle} className="fa-lg" title="help menu"/>
+              </a>
+              <div className="dropdown-menu dropdown-menu-end p-3">
+                <ul className="list-unstyled">
+                  <li>
+                    <Link className="dropdown-item" to="/help" target="_blank">Help pages</Link>
```
Comment (MatthewBemis) [inline ui-admin/src/navbar/AdminNavbar.tsx:70]: The help page dropdown is visible but not clickable when a user is not logged in. Should this be clickable?

I could imagine the 'Contact support' option being useful for a logged out user if they're having login issues.

**Review by MatthewBemis — APPROVED** [review]
One question, but other than that 👍

**Comment by breilly2** [comment]
I'm seeing an unnecessary horizontal scrollbar to accommodate about the width of a scrollbar.
Screenshot 2023-07-31 at 11 45 45 AM
To reproduce, make sure the macOS system preference "Appearance > Show scroll bars" is set to "Always" (or possibly "Automatically based on mouse or trackpad" depending on your peripherals).

Edit: This is not a new bug, so I'm not going to hold it against this PR.

**Review by breilly2 — APPROVED** [review]

File: ui-admin/src/navbar/AdminNavbar.tsx (hunk 1 above)
Comment (devonbush) [reply ui-admin/src/navbar/AdminNavbar.tsx:70]: yeah, currently the login modal blocks everything.  We'll probably want to just add a separate "help with login?" link inside the modal

**Pull request merged**